docs/stories/alert.stories.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import styled from 'react'
+import {storiesOf} from '@storybook/react'
+import {palette, Info, VslIcon, CheckIcon, WarningIcon, ForbiddenIcon, QuestionIcon, Typography, Alert} from '@ambler/andive'
+
+import Showcase from './showcase'
+
+
+storiesOf('NEXT|Alert', module)
+  .add('Variant', () => {
+    return (
+      <>
+        <Showcase legend="Variant">
+          <Alert>
+            <Typography.Body3>Default</Typography.Body3>
+          </Alert>
+          <Alert variant="success">
+            <Typography.Body3>Success</Typography.Body3>
+          </Alert>
+          <Alert variant="error">
+            <Typography.Body3>Error</Typography.Body3>
+          </Alert>
+          <Alert variant="warning">
+            <Typography.Body3>Warning</Typography.Body3>
+          </Alert>
+          <Alert variant="info">
+            <Typography.Body3>Info</Typography.Body3>
+          </Alert>
+          <Alert variant="disabled">
+            <Typography.Body3>Disabled</Typography.Body3>
+          </Alert>
+        </Showcase>
+        <Showcase legend="Icon">
```

Comment:
Instead of 1 story, make a story per prop:
```
.add("Variant", () => <>
  <Showcase legend="success">
    <Alert variant="success">...</Alert>
  </Showcase>
  <Showcase legend="error">
    <Alert variant="error">...</Alert>
  </Showcase>
  ...
</>)
.add("With an Icon", () => <>
  <Showcase legend="success">
    <Alert variant="success">
      <WarningIcon/>
      <Typography.Body3>Default</Typography.Body3>
    </Alert>
  </Showcase>
  ...
</>)
...
```